src/D2L.Security.OAuth2/Keys/Default/D2LSecurityToken.cs

```diff
@@ -1,15 +1,11 @@
 ﻿using System;
-using System.IdentityModel.Tokens;
 using System.Security.Cryptography;
 using System.Threading;
+using Microsoft.IdentityModel.Tokens;
 
 namespace D2L.Security.OAuth2.Keys.Default {
 
-	internal sealed partial class D2LSecurityToken : SecurityToken {
-
-		private string m_id;
-		private readonly DateTimeOffset m_validFrom;
-		private readonly DateTimeOffset m_validTo;
+	internal sealed partial class D2LSecurityToken : SecurityKey {

```

Comment:
This should get renamed `D2LSecurityKey` for less confusion, but trying not to bloat the diff more.